doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -712,17 +719,17 @@ or:
 I Reproduced the problem with appropriate debug levels. Now what?
 -----------------------------------------------------------------
 
-We prefer that you send us only the portions of your logs that are relevant to
-your monitor problems. Of course, it might not be easy for you to determine
-which portions are relevant so we are willing to accept complete and
-unabridged logs. However, we request that you avoid sending logs containing
-hundreds of thousands of lines with no additional clarifying information. One
-common-sense way of making our task easier is to write down the current time
-and date when you are reproducing the problem and then extract portions of your
-logs based on that information.
-
-Finally, reach out to us on the mailing lists or IRC or Slack, or by filing a
-new issue on the `tracker`_.
+Send the upstream Ceph community only the portions of your logs that are
```

Comment:
S/upstream//

Just one community, so we don’t need to qualify?

Reply:
I want to distinguish the upstream community  from communities for which a person might pay for support.

Reply:
I wouldn't call the latter a _community_ but you're the boss, head man, top dog, big cheese, the head honcho, number...

Reply:
I chose the wrong week to stop sniffing glue.